api/src/test/java/org/pmiops/workbench/db/model/StorageEnumsTest.java

```diff
@@ -75,16 +75,16 @@ private Object storageToEnum(Short shortValue, Class enumClass)
   }
 
   /**
-   * Retrieve all Enum classes in DbStorageEnums and CommonStorageEnums
+   * Retrieve all Enum classes in DbStorageEnums and DbStorageEnums
```

Comment:
Nit: this comment should be revised to just `DbStorageEnums`

Reply:
Looooooool. I did a global stringy search/replace. This is what I get. Thank you!

Reply:
Lines 23 and 87 have similar dupes

Reply:
And here's a dumb little PR to fix https://github.com/all-of-us/workbench/pull/3259